cmd/codeqlExecuteScan.go

```diff
@@ -227,6 +229,15 @@ func runCodeqlExecuteScan(config *codeqlExecuteScanOptions, telemetryData *telem
 
 	reports = append(reports, piperutils.Path{Target: fmt.Sprintf("%vtarget/codeqlReport.csv", config.ModulePath)})
 
+	// create toolrecord file
+	toolRecordFileName, err := createToolRecordCodeql(utils, "./", *config)
```

Comment:
Could you do this after uploading the results? As the links in the tool record do not make sense. It makes sense only after uploading sarif files.